app/Services/BaseService.php

```diff
@@ -139,6 +139,29 @@ public function canBypassPermissionLevelIfEmployee(int $employeeId): self
         return $this;
     }
 
+    /**
+     * Sets the permission to bypass the minimum level requirement necessary to
+     * execute the service if the author who calls it is actually the employee
+     * or the manager.
+     *
+     * @param int $employeeId
+     * @return self
+     */
+    public function canBypassPermissionLevelIfEmployeeOrManager(int $managerId, int $employeeId): self
+    {
+        $manager = Employee::where('company_id', $this->companyId)
+            ->findOrFail($managerId);
+
+        $isManager = $manager->isManagerOf($employeeId);
+        $this->bypassRequiredPermissionLevel = $isManager;
+
+        if ($this->bypassRequiredPermissionLevel == false) {
+            $this->bypassRequiredPermissionLevel = ($this->authorId == $employeeId);
```

Comment:
use `===` instead of `==`
```suggestion
        if ($this->bypassRequiredPermissionLevel === false) {
            $this->bypassRequiredPermissionLevel = ($this->authorId === $employeeId);
```